### PR TITLE
#516 Removed all sanitized text from renderable arrays in favor of built in sanitization, and remove unused code, and mark inlined svg as experimental.

### LINKED ISSUE #516: Address remaining theme logo issues.
- [ ] remove theme config settings from the info file.
- [ ] sanitize alt and title text only once per function.
- [ ] mark inline svg as experimental.

### PR DESCRIPTION
**Pull request by trackleft**
## Related Issue
Closes #516 
## How Has This Been Tested?
Tested locally, but would like a second opinion
view-source:https:::localhost:55034 2021-02-25 16-55-43
Arizona Barrio | Drush Site-Install 2021-02-25 16-54-50

Tested with script tags, seems to sanitize output
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: themes/custom/az_barrio/az_barrio.info.yml [hunk 1]
```diff
@@ -3,8 +3,6 @@ type: theme
 description: 'Basic structure for a Bootstrap Barrio SubTheme.'
 core_version_requirement: ^8.8 || ^9
 base theme: bootstrap_barrio
-logo: images/logo.png
-footer_logo: images/logo.png
```
Comment (joeparsons) [inline themes/custom/az_barrio/az_barrio.info.yml:7]: Should we remove the actual PNG image file from the repo as well?
https://github.com/az-digital/az_quickstart/blob/issue/516-remaining-logo-issues/themes/custom/az_barrio/images/logo.png

**Comment by joeparsons** [comment]
I think you can remove this `use` statement as well:
```
use Drupal\Component\Utility\Html;
```
https://github.com/az-digital/az_quickstart/blob/71594737f57ed68d329e729c03b0c423ee93b8f0/themes/custom/az_barrio/az_barrio.theme#L13

**Review by joeparsons — COMMENTED** [review]
See my other comments.  Overall I think this looks good.

File: themes/custom/az_barrio/az_barrio.info.yml (hunk 1 above)
Comment (trackleft) [reply themes/custom/az_barrio/az_barrio.info.yml:7]: We still use the logo here: https://github.com/az-digital/az_quickstart/blob/main/themes/custom/az_barrio/config/install/az_barrio.settings.yml#L68

**Review by joeparsons — APPROVED** [review]

**Review by jwmoore1 — APPROVED** [review]

**Comment by kevcooper** [comment]
Tried various methods of injecting new attributes into the img tag and was not successful - Drupal's quote encoding seems to be working as expected!

**Review by kevcooper — APPROVED** [review]

**Pull request merged**